static/app/components/eventOrGroupHeader.tsx

```diff
@@ -104,38 +108,37 @@ class EventOrGroupHeader extends Component<Props> {
               eventID ? `events/${eventID}/` : ''
             }`,
             query: {
-              query: this.props.query,
-              ...(location.query.sort !== undefined ? {sort: location.query.sort} : {}), // This adds sort to the query if one was selected from the issues list page
-              ...(location.query.project !== undefined ? {} : {_allp: 1}), // This appends _allp to the URL parameters if they have no project selected ("all" projects included in results). This is so that when we enter the issue details page and lock them to a project, we can properly take them back to the issue list page with no project selected (and not the locked project selected)
+              query,
+              ...(loc.query.sort !== undefined ? {sort: loc.query.sort} : {}), // This adds sort to the query if one was selected from the issues list page
```

Comment:
Should we find a better alias name for `location`? My mind jumped to `Lines Of Code`.

Reply:
location is already being used and I did not want to change that.. but I can use `props.location `